### ISSUE TITLE
[opencascade] build failure

### ISSUE THREAD

**Issue by Bzz2745** [microsoft/vcpkg#33737]
### Operating system

Windows 11

### Compiler

MSVC vc143

### Steps to reproduce the behavior

```Shell
`vcpkg install opencascade[freeimage,rapidjson,samples,tbb]:x64-windows --clean-after-build`
```

### Failure logs

Computing installation plan...
The following packages will be built and installed:
    opencascade[core,freeimage,rapidjson,samples,tbb]:x64-windows -> 7.6.2#7
Detecting compiler hash for triplet x64-windows...
Restored 0 package(s) from C:\Users\BaZunzhi\AppData\Local\vcpkg\archives in 469 us. Use --debug to see more details.
Installing 1/1 opencascade:x64-windows...
Building opencascade[core,freeimage,rapidjson,samples,tbb]:x64-windows...
-- Using cached Open-Cascade-SAS-OCCT-bb368e271e24f63078129283148ce83db6b9670a.tar.gz.
-- Extracting source C:/vcpkg/downloads/Open-Cascade-SAS-OCCT-bb368e271e24f63078129283148ce83db6b9670a.tar.gz
-- Applying patch fix-pdb-find.patch
-- Applying patch fix-install-prefix-path.patch
-- Applying patch install-include-dir.patch
-- Applying patch fix-depend-freetype.patch
-- Applying patch fix-dependence.patch
-- Using source at C:/vcpkg/buildtrees/opencascade/src/3db6b9670a-0dce2cc883.clean
-- Found external ninja('1.11.0').
-- Configuring x64-windows
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: "C:/Program Files/Microsoft Visual Studio/2022/Community/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" -v
    Working Directory: C:/vcpkg/buildtrees/opencascade/x64-windows-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      C:\vcpkg\buildtrees\opencascade\config-x64-windows-dbg-CMakeCache.txt.log
      C:\vcpkg\buildtrees\opencascade\config-x64-windows-rel-CMakeCache.txt.log
      C:\vcpkg\buildtrees\opencascade\config-x64-windows-out.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:252 (vcpkg_execute_required_process)
  ports/opencascade/portfile.cmake:34 (vcpkg_cmake_configure)
  scripts/ports.cmake:147 (include)

### Additional context

[config-x64-windows-rel-CMakeCache.txt.log](https://github.com/microsoft/vcpkg/files/12595624/config-x64-windows-rel-CMakeCache.txt.log)
[config-x64-windows-dbg-CMakeCache.txt.log](https://github.com/microsoft/vcpkg/files/12595625/config-x64-windows-dbg-CMakeCache.txt.log)
[config-x64-windows-out.log](https://github.com/microsoft/vcpkg/files/12595626/config-x64-windows-out.log)

**Comment by Bzz2745** [comment]
I have already installed tbb together with oneapi.

**Comment by FrankXie05** [comment]
Will be fixed by PR https://github.com/microsoft/vcpkg/pull/33376

**Comment by waywardmonkeys** [comment]
Since #33376 was a duplicate of #33498 which did the update of opencascade, there was also a build failure reported in #27896 which was fixed by #33498. Is this fixed now then? (cc: @Adela0814 @FrankXie05)

**Comment by FrankXie05** [comment]
@waywardmonkeys It bas been fixed.
